cmd/security-agent/app/runtime.go

```diff
@@ -129,5 +130,8 @@ func startRuntimeSecurity(hostname string, endpoints *config.Endpoints, context
 
 	log.Info("Datadog runtime security agent is now running")
 
+	// Send the runtime 'running' metrics periodically
+	go secagentcommon.SendRunningMetrics("runtime")
```

Comment:
The `Ticker` started in the goroutine implements the `Stop` method, so it can be added to the `stopper` object to stop properly

Reply:
Done.